src/test/java/com/revature/services/test/AnswerServiceTest.java

```diff
@@ -44,11 +44,13 @@
 	@Autowired
 	RSSService rssservice;
 	
+	private Date ts = new Date(100000);
+
 	/** @author ken */
 	@Test
 	public void getAllAnswersTest() throws Exception {
 		
-		Answer answer = new Answer(1, 1, 1, "test content", LocalDate.MIN, LocalDate.MIN);	
+		Answer answer = new Answer(1, 1, 1, "test content", ts, ts);	
```

Comment:
We should not be using `java.sql.Date` unless you are conducting integration tests with the Database that necessitate using it. Use LocalDate for all internal application date handling.

Reply:
fixed